staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -58,6 +58,12 @@ type Storage interface {
 	New() runtime.Object
 }
 
+// Destroy indicates a RESTful storage is capable of cleaning its allocated resources.
+type Destroy interface {
```

Comment:
Destroy -> Destroyer

Reply:
Was trying to avoid the name. Neither Terminator caught my eye.

Reply:
How about the following:
```
type Closeable interface {
    Close()
}
```

Reply:
`registry.Store` itself has `DestroyFunc` field and the word `Destroy` is used all over the place. Picking `Destroyable` for now. Though still open for changes.